src/enums/zone_constants.ts

```diff
@@ -0,0 +1,35 @@
+import { COLOR_MAP } from 'enums/interventions';
+
+export interface LevelInfo {
+  level: string;
+  upperLimit: number;
+  name: string;
+  color: string;
+  detail: string;
+}
+
+export enum Level {
+  LOW = 'LOW',
+  MEDIUM = 'MEDIUM',
+  HIGH = 'HIGH',
+  UNKNOWN = 'UNKNOWN',
+}
+
+export const COLOR_ZONE = {
+  [Level.LOW]: COLOR_MAP.GREEN.BASE,
+  [Level.MEDIUM]: COLOR_MAP.ORANGE.BASE,
+  [Level.HIGH]: COLOR_MAP.RED.BASE,
+  [Level.UNKNOWN]: COLOR_MAP.GRAY.BASE,
+};
+
+export interface Zones {
+  [Level.LOW]: LevelInfo;
+  [Level.MEDIUM]: LevelInfo;
+  [Level.HIGH]: LevelInfo;
+  [Level.UNKNOWN]: LevelInfo;
+}
+
+export const LOW_NAME = 'Reduced';
+export const MEDIUM_NAME = 'Moderate';
+export const HIGH_NAME = 'Elevated';
+export const UNKNOWN = 'Unknown';
```

Comment:
Just as a note, in the first designs, each chart had a different label for each level. Currently, the labels for the charts are High, Medium and Low, consider changing these back either on the metric files or here (unless there is an explicit decision to change them to Elevated, Moderate, and Reduced).

Reply:
yeah okay this is now still allowable